src/components/Login.css

```diff
@@ -0,0 +1,69 @@
+.login-container {
+    background: white;
+    padding: 60px 30px;
+    border-radius: 8px;
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+    text-align: center;
+    width: 300px;
+}
+
+h2 {
+    margin-bottom: 20px;
+}
+
+.input-group {
+    margin: 10px 0;
+    text-align: left;
+}
+
+.input-group label {
+    display: block;
+    margin-bottom: 5px;
+}
+
+.input-group input {
+    width: 100%;
+    padding: 8px;
+    box-sizing: border-box;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+}
+
+.remember-group {
+    display: flex;
+    align-items: center;
+    margin: 10px 0;
+}
+
+.remember-group label {
+    margin-left: 5px;
+}
+
+.button-group {
+    display: flex;
+    justify-content: space-between;
+}
+
+.login-btn,
+.home-btn {
+    padding: 10px;
+    width: 45%;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+}
+
+.login-btn {
+    background-color: #3498db;
```

Comment:
For now, let's keep the color codes as they are, but in the future, we will need to manage them centrally through a file like variable.css. This implementations are like a sketch .